src/me/mrCookieSlime/Slimefun/listeners/BackpackListener.java

```diff
@@ -41,27 +45,41 @@ public void onClose(InventoryCloseEvent e) {
 	}
 	
 	@EventHandler
+	public void onItemDrop(PlayerDropItemEvent e) {
+		if (Variables.backpack.containsKey(e.getPlayer().getUniqueId())){
+			ItemStack item = e.getItemDrop().getItemStack();
+			SlimefunItem sfItem = SlimefunItem.getByItem(item);
+			if (sfItem instanceof SlimefunBackpack || sfItem instanceof SoulboundBackpack) e.setCancelled(true);
```

Comment:
SoulboundBackpack extends SlimefunBackpack, so it is not needed to check for this.